customers/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+
+# Create your models here.
+class Person(models.Model):
+    name = models.CharField(max_length=20, default="Cami")
```

Comment:
You should drop the default from the fields. Also some of those fields might be required and some not. Read [this article](https://sentry.io/answers/django-difference-between-null-and-blank/) to get a better understanding of `null` and `blank` proprieties.